apis/core/lib/domain/csv-mapping/delete.ts

```diff
@@ -21,6 +21,13 @@ export async function deleteMapping(csvMapping: NamedNode, store: ResourceStore)
     store.delete(sourceCollection)
   }
 
+  const tables = getTablesFromMapping(csvMapping)
+  for await (const table of tables) {
+    if (table.termType === 'NamedNode') {
+      await deleteTableWithoutSave(table, store)
+    }
```

Comment:
I'd move the check inside the delete function (no-op if not named node)

```suggestion
    await deleteTableWithoutSave(table, store)
```

The parameter would become `Term`

Reply:
Should I do the same thing with the sources?